### ISSUE TITLE
Sashimi plot for paired-end reads

### ISSUE THREAD

**Issue by cnzqy1** [igvteam/igv#1146]
I am working with sequencing data that has short insert sizes but was sequenced with 150 bp paired-end reads, leading to high levels of overlaps within the read pairs. When I visualized the data aligned by STAR in IGV with sashimi plot, I got the following:

![Sashimi](https://user-images.githubusercontent.com/8440715/170903702-ca65bf4e-dd62-4119-b73c-f160081cb310.png)

Which comes from the following reads:

![read pairs](https://user-images.githubusercontent.com/8440715/170903729-45471c5e-3ec9-4d1f-ac0b-725930a22338.png)

However, you can see that these 4 reads come from 2 read pairs with 100% overlap within read pairs. Wouldn't counting the number of read pairs be more informative than counting the number of reads? In fact, in `SJ.out.tab` from STAR output, read pairs are counted (2 in this particular case).

**Comment by jrobinso** [comment]
Sorry I don't understand the point you are making.

**Comment by jrobinso** [comment]
No its not.   Everything is possible of course, no ETA on this as you can see from the number of open issues there is a lot on my plate.

**Comment by jrobinso** [comment]
Yes understood, just pointing out that overlapping read pairs from short insert sizes cause other issues in analysis and visualization.   These are technically not duplicates but effectively so, at least the overlapping part.
